package.json

```diff
@@ -58,6 +59,8 @@
     }
   },
   "dependencies": {
+    "@typescript-eslint/scope-manager": "^8.46.1",
+    "@typescript-eslint/visitor-keys": "^8.46.1",
```

Comment:
I don't think we need these dependencies, or we should move them to devDeps, since we're only using them for type checking.